### DISCUSSION TITLE
Installation interrupted -- do I need to reinstall?

### DISCUSSION THREAD

**Discussion by swiesenfeld0** [oschwengers/bakta#145] [Q&A] [answered]
Hi,

I'm installing bakta for the first time and it got interrupted at the very end. Below is the log. Do I need to redo this from the beginning download command? Is there a way I can resume whatever it needed to do without redownloading the database?

`(assembler) [swiesenfeld@holy2a23303 sophia]$ bakta_db download --output <path>
Bakta software version: 1.5.0
Required database schema version: 4

fetch DB versions...
        ... compatible DB versions: 1
download database: v4.0, 2022-08-29, DOI: 10.5281/zenodo.7025248, URL: https://zenodo.org/record/7025248/files/db.tar.gz...
|████████████████████████████████████████| 31734559/31734559 [100%] in 51:28.7 (10274.25/s)
        ... done
check MD5 sum...
        ...database file OK: 4116192902e88140b19b5e4d88be9fa1
extract DB tarball: file=/n/holylfs05/LABS/hanage_lab/Lab/holyscratch01/sophia/baktadb/db.tar.gz, output=/n/holylfs05/LABS/hanage_lab/Lab/holyscratch01/sophia/baktadb
salloc: Job allocation 26458782 has been revoked.
                                                 slurmstepd: error: *** STEP 26458782.interactive ON holy2a23303 CANCELLED AT 2022-09-14T11:57:19 DUE TO PREEMPTION ***
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
srun: error: holy2a23303: task 0: Killed`

**Comment by oschwengers** [comment, marked as answer]
Hi @swiesenfeld0, in this case could see if the downloaded file `n/holylfs05/LABS/hanage_lab/Lab/holyscratch01/sophia/baktadb/db.tar.gz` is still in place.

If this is the case, you should be able to conduct the remaining installation procedure manually as described in the last part of the readme/installation section (https://github.com/oschwengers/bakta#installation):
```bash
tar -xzf db.tar.gz
amrfinder_update --force_update --database db/amrfinderplus-db/
```

I hope this saves another download.
